### PR TITLE
Cleanup/copyrights and empty events

### PR DESCRIPTION
**Pull request by robszewczyk**

### CODE REVIEW

File: codegen/weave-device-cpp/templates/event_struct.cpp.inc [hunk 1]
```diff
@@ -64,4 +64,12 @@ const nl::SchemaFieldDescriptor {{struct.base_name}}::FieldSchema =
     .mFields = {{struct.base_name}}FieldDescriptors,
     .mSize = sizeof({{struct.base_name}})
 };
+{% else %}
```
Comment (gerickson) [inline codegen/weave-device-cpp/templates/event_struct.cpp.inc]: Gratuitous, unrelated change?

File: codegen/weave-device-cpp/reference/nest/test/trait/TestETrait.h [hunk 2]
```diff
@@ -511,6 +511,26 @@ inline bool TestENullableEvent::IsNeJPresent(void)
 }
 #endif
 
+struct TestEEmptyEvent
```
Comment (gerickson) [inline codegen/weave-device-cpp/reference/nest/test/trait/TestETrait.h]: Gratuitous, unrelated change?

File: codegen/weave-device-cpp/reference/nest/test/trait/TestETrait.cpp [hunk 3]
```diff
@@ -242,6 +242,22 @@ const nl::Weave::Profiles::DataManagement::EventSchema TestENullableEvent::Schem
     .mMinCompatibleDataSchemaVersion = 1,
 };
 
+const nl::SchemaFieldDescriptor TestEEmptyEvent::FieldSchema =
```
Comment (gerickson) [inline codegen/weave-device-cpp/reference/nest/test/trait/TestETrait.cpp]: Gratuitous, unrelated change?

File: test/schema/nest/test/trait/test_e_trait.proto [hunk 4]
```diff
@@ -148,4 +167,15 @@ message TestETrait {
         google.protobuf.StringValue ne_i = 9 [(wdl.param) = {nullable: true}];
         NullableE ne_j = 10 [(wdl.param) = {nullable: true}];
     }
+
+    message TestEEmptyEvent {
```
Comment (gerickson) [inline test/schema/nest/test/trait/test_e_trait.proto]: Gratuitous, unrelated change?

File: codegen/weave-device-cpp/templates/event_struct.cpp.inc (hunk 1 above)
Comment (robszewczyk) [reply codegen/weave-device-cpp/templates/event_struct.cpp.inc]: Three commits tackling two unrelated issues.  I had named the PR to be descriptive of both changes, and the commit messages are meaningful and relate to the right sections of the functionality.  Unfortunately, the resulting PR ends up ignoring the commit messages in the top-level description.  I will split this into different PRs.

Comment (robszewczyk) [reply codegen/weave-device-cpp/templates/event_struct.cpp.inc]: Should be resolved now.
